frontend/schemaCMS/src/routes/dataSource/dataSource.component.js

```diff
@@ -91,7 +91,7 @@ export default class DataSource extends PureComponent {
     const sourcePath = `${path}/source`;
     const jobListPath = `${path}/job`;
     const hasActiveJob = pathEq(['dataSource', 'activeJob'], null, this.props);
-    const projectId = pathOr('', ['project', 'id'], dataSource);
+    const projectId = pathOr(dataSource.project || '', ['project', 'id'], dataSource);
```

Comment:
We shouldn't mix ternary operators and ramda like that. 
```
 const projectId = pathOr(propOr('', 'project', dataSource), ['project', 'id'], dataSource);
```

Reply:
I don't see anything wrong with that implementation, sometimes to simplest things we are always forced to use ramda what for me is wrong.

Reply:
The point is if dataSource is undefined for some reason? And it doesn't make sense if you use `pathOr` and then `||`... the idea of `pathOr` (and `propOr`) is to have a descriptive way of doing this. It's up to you, but I vote for consistency :)